source/_integrations/fluss.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: Fluss+
+description: Instructions on how to integrate Fluss+ devices within Home Assistant.
+ha_category:
+  - Button
+ha_release: 2024.5
+ha_iot_class: Cloud Polling
+ha_codeowners:
+    -'@fluss'
+ha_domain: fluss
+ha_config_flow: true
+ha_platforms:
+  - button
+ha_integration_type: integration
+related:
+  - docs: /docs/configuration/
+    title: Configuration file
+---
+
+The `Fluss+` integration allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.
```

Comment:
```suggestion
The **Fluss+** {% term integrations %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.
```
For items listed in the [glossary](/docs/glossary/), we have these tooltips to show the definition.
![image](https://github.com/user-attachments/assets/e46061bf-2a94-46ba-bffa-32c058005653)